### ISSUE TITLE
NVDA- When editing a date by pressing Enter on a DatePicker textfield, date is announced twice, once as selected and once as not selected.

### ISSUE THREAD

**Issue by shoegal310** [OfficeDev/office-ui-fabric-react#9186]
## Codepen demonstrating the issue
https://codepen.io/shoegal310/pen/yWpwWR?&editable=true

I have a datepicker control in which I select a date.
Now I go back to the control and press enter to open the datepicker

## Actual behavior:
Screen reader announces the selected date first as "selected" and then as "unselected" along with rest of row and column information
## Expected behavior:
Screen reader should announce the date only once and as selected

**Comment by kenotron** [comment]
This behavior doesn't happen on Edge + Narrator. We suspect this to be an incorrect behavior on NVDA's part.
